### PR TITLE
Update to hyper 1.0 in order to be able to add requested features.

### PR DESCRIPTION
**Pull request by DFINITYManu**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -19,6 +18,8 @@ pub async fn run() {
     }
     let mut set = JoinSet::new();
 
+    simple_logger::init_with_level(log::Level::Trace).unwrap();
```
Comment (sasa-tomic) [inline src/main.rs:21]: Do we need Trace in production?

File: src/proxy.rs [hunk 2]
```diff
@@ -29,38 +33,74 @@ static STRIP_FROM_RESPONSE: [&str; 1] = ["content-length"];
...
+                    }
+                    // Ignore such headers;
+                    Err(err) => {
+                        error!("Invalid request header value: {}", err)
+                    }
+                }
             }
         }
     }
     // println!("Filtered: {:?}", headers);
```
Comment (sasa-tomic) [inline src/proxy.rs:98]: drop?

**Review by sasa-tomic — APPROVED** [review]
Very nice, thanks!

File: src/main.rs (hunk 1 above)
Comment (DFINITYManu) [reply src/main.rs:21]: Good catch.

**Pull request merged**